### ISSUE TITLE
Missing accessibility label on edit icon to change wallet name 

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1455]
### Description of problem
Missing accessibility label on edit icon to edit wallet name 

### Expected behavior
When focused should describe the icon button as "edit wallet name"

### Steps to reproduce
- turn on voice reader in OS settings
- go to menu
- swipe to "edit" icon next to wallet name

### Screenshots and/or log output
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbmlRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--e71bccd88057ffd2d7ad65404afdf4e4ab161e81/20230912_094922.png)

### Environment
- [X] Occurs on Android
- [X] Occurs on iOS

Build #: 1193

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [X] Medium
- [ ] Low

**Comment by knguyenBC** [comment]
Checked both android and ios. Looks good.
